testplans/lkft-sanity/ltp-quickhit.yaml

```diff
@@ -1 +0,0 @@
-../../testcases/ltp-quickhit.yaml
```

Comment:
Don't remove this from here either right now.
Others depends on it.
That may be wrong but that is how it is right now sadly.
Until that is fixed we have to leave this file here.
We will still be able to move over to ltp-smoketest for our new rootfs.

Reply:
@roxell Thanks for the review.
Your review comments have been addressed.